apps/site/src/components/pages/wordpress/header.tsx

```diff
@@ -0,0 +1,150 @@
+import { Stream } from "@cloudflare/stream-react";
+import { Box, Container, Typography, useTheme } from "@mui/material";
+import Image from "next/legacy/image";
+
+import wpHelixImage from "../../../../public/assets/new-home/wp-helix.png";
+import { BpWpIcons } from "./bp-wp-icons";
+import { EarlyAccessCTA } from "./early-access-cta";
+
+export const Header = () => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        pb: { xs: 20, md: 28 },
+        position: "relative",
+      }}
+    >
+      <Container
+        sx={{
+          position: "relative",
+          zIndex: 3,
+          pt: { xs: 16, md: 20 },
+          mb: { xs: 6, md: 10 },
+          maxWidth: { xs: "95%", md: "75%", lg: "60%" },
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+        }}
+      >
+        <Typography
+          sx={{
+            textTransform: "uppercase",
+            color: ({ palette }) => palette.purple[700],
+            mb: 2.25,
+            fontWeight: 500,
+            letterSpacing: "0.06rem",
+            mx: "auto",
+          }}
+          textAlign="center"
+          variant="bpSmallCaps"
+        >
+          EARLY ACCESS
```

Comment:
Where we use all-caps, we want to style the text via CSS (`text-transform: uppercase`), but keep the original in-code string reading normally (i.e. this would be "Early access")

Reply:
Addressed in c5668ea8e7a7f95f0b3f8b312d857d9fc6aaa7e0